api/room/schema_query.py

```diff
@@ -405,15 +405,17 @@ def resolve_bookings_analytics_count(
             self, query, start_date, end_date, room_id=room_id)
         return analytics
 
+    @Auth.user_roles('Admin', 'Default User')
     def resolve_analytics_for_daily_room_events(
         self, info, start_date, end_date, page=None, per_page=None
```

Comment:
something like this should do the trick. @daniellamarr 
```suggestion
        self, info, **kwargs
```

Reply:
I have made the changes